scripts/trace_flowcell.py

```diff
@@ -0,0 +1,152 @@
+import os
+import requests
+import json
+import sys
+import subprocess
+import yaml
+import argparse
+from datetime import datetime
+
+class GetTraces:
+    
+    def __init__(self, api_base_url, access_headers, req_verify, workflow):
+        self.api_base_url = api_base_url
+        self.headers = access_headers
+        self.verify = req_verify
+        self.workflow = workflow
+    
+    def get_executions_for_tag(self, tag):
+        traces_response = requests.get(
+            "{}/{}/?{}={}".format(self.api_base_url, "traces", "trace_tag", tag),
+            verify = self.verify,
+            headers = self.headers)
+        if not traces_response.ok:
+            raise Exception("Response not OK, got: " + traces_response.text)
+        traces = json.loads(traces_response.text)
+        for trace in traces:
+            trace_id = trace["id"]
+            trace_info_response = requests.get(
+                "{}/{}/{}".format(self.api_base_url, "traces", trace_id),
+                verify = self.verify,
+                headers = self.headers)
+            trace_info = json.loads(trace_info_response.text)
+            action_executions = map(lambda x: x["object_id"], trace_info["action_executions"])
+            for execution in action_executions:
+                yield execution
+
+    def get_actions_from_executions(self, executions):
+        for execution_id in executions:
+            execution_info_response = requests.get(
+                "{}/{}/{}".format(self.api_base_url, "executions", execution_id),
+                verify = self.verify,
+                headers = self.headers)
+            execution_info = json.loads(execution_info_response.text)
+            yield execution_info
+    
+    def filter_and_order_actions_by_name(self, actions):
```

Comment:
wouldn't it make more sense to name this function `filter_and_order_actions_by_date`?